src/simulation.hpp

```diff
@@ -2094,7 +2091,10 @@ void AMRSimulation<problem_t>::AverageFCToCC(amrex::MultiFab &mf_cc, const amrex
 	// iterate over the domain
 	auto const &state_cc = mf_cc.arrays();
```

Comment:
This is not the global state multifab. This is just whatever multifab is passed in as an argument to this function as `mf_cc`.

Reply:
@markkrumholz This variable is confusingly named. I think this is the source of the confusion.

Reply:
I see. OK, thanks for clarifying. In that case I'm happy to approve the PR.